src/ui/components/UserReview/styles.scss

```diff
@@ -25,10 +30,11 @@
 
 .UserReview-body {
   font-size: $font-size-default;
+  margin: 0 !important;
   word-wrap: break-word;
 
   &:not(.UserReview-emptyBody) {
-    margin: 4px 0 6px;
+    margin: 4px 0 6px !important;
```

Comment:
We don't tend to use `!important`. Why is it needed here?

Reply:
Same here after adding `ShowMoreCard` then `UserReview-body` class margin get conflict with 
`.Addon .Card`.